Tests/Invoke-Command.Tests.ps1

```diff
@@ -0,0 +1,55 @@
+Import-Module -Force $PSScriptRoot/../Docker.Build.psm1
+. "$PSScriptRoot\..\Private\Invoke-Command.ps1"
+. "$PSScriptRoot\..\Private\CommandResult.ps1"
+
+Describe 'Run external tools as commands' {
+
+    Context 'Run a simple external command'  {
+        if ($IsWindows) {
+            $commandName = "find /?"
```

Comment:
`find /?` will work in batch. Find is also a git bundled utility. 
However, I would like this to succeed in PowerShell prompt as well. It's not a default alias so far.

Reply:
Invoke-Command works with both CmdLets and external programs. Using find is on purpose, so I can test that running external programs work. If I used 'dir' for instance it would invoke the CmdLet given that there is an alias for 'dir'.
There is another test case which covers running CmdLets so we should be good.